.circleci/config.yml

```diff
@@ -220,7 +220,7 @@ jobs:
       WHEELHOUSE_PATH: &WHEELHOUSE_PATH "/tmp/wheelhouse"
       PIP_FIND_LINKS: "file:///tmp/wheelhouse"
       # Upload the coverage report.
-      UPLOAD_COVERAGE: "yes"
+      UPLOAD_COVERAGE: ""
```

Comment:
So, this is never used now, right? Can we just delete the "submit coverage" job / stanza that this enabled too (or is that premature for some reason)?

Reply:
I suppose I am reluctant to do so because I think it is more likely we will eventually be all CircleCI than that we will eventually be all GitHub Actions so I imagine re-enabling CircleCI coverage upload at some point.